plugins/GSdx/GSRendererDX11.cpp

```diff
@@ -37,10 +43,54 @@ bool GSRendererDX11::CreateDevice(GSDevice* dev)
 	return true;
 }
 
-void GSRendererDX11::SetupIA()
+void GSRendererDX11::SetupIA(const float& sx, const float& sy)
 {
 	GSDevice11* dev = (GSDevice11*)m_dev;
 
+	D3D11_PRIMITIVE_TOPOLOGY t;
+
+	bool unscale_hack = UserHacks_unscale_pt_ln  & (GetUpscaleMultiplier() > 1);// &GLLoader::found_geometry_shader;
```

Comment:
I just copied that line from Gregory's OGL code. As there's no custom resolution on Linux anymore, he probably didn't even consider it.

I thought I'd removed the comment and double space though. :S